sdcm/mgmt/common.py

```diff
@@ -24,6 +26,53 @@ def duration_to_timedelta(duration_string):
     return datetime.timedelta(seconds=total_seconds)
 
 
+def get_manager_repo_from_defaults(manager_branch_name, distro):
+    with open("defaults/manager_versions.yaml", 'r') as mgmt_config:
+        manager_repos_by_branch_dict = yaml.load(mgmt_config, Loader=yaml.BaseLoader)["manager_repos_by_branch"]
+
+    branch_specific_repos = manager_repos_by_branch_dict.get(manager_branch_name, None)
+    assert branch_specific_repos, f"Couldn't find manager branch {manager_branch_name} in manager defaults"
+
+    known_distro_dict = {Distro.CENTOS7: "centos7",
+                         Distro.DEBIAN9: "debian9",
+                         Distro.DEBIAN10: "debian10",
+                         Distro.UBUNTU16: "ubuntu16",
+                         Distro.UBUNTU18: "ubuntu18",
+                         Distro.UBUNTU20: "ubuntu20"}
+
+    distro_name = known_distro_dict.get(distro, None)
+    assert distro_name, f"Unfamiliar distro: {distro}"
+
+    repo_address = branch_specific_repos.get(distro_name, None)
+    assert repo_address, f"Could not find manager repo for distro {distro_name} in branch {manager_branch_name}"
+
+    return repo_address
+
+
+def get_manager_scylla_backend(scylla_backend_branch_name, distro):
+    with open("defaults/manager_versions.yaml", 'r') as mgmt_config:
+        scylla_backend_repos_by_branch_dict = yaml.load(mgmt_config,
+                                                        Loader=yaml.BaseLoader)["scylla_backend_repo_by_branch"]
+
+    branch_specific_repos = scylla_backend_repos_by_branch_dict.get(scylla_backend_branch_name, None)
+    assert branch_specific_repos, f"Couldn't find scylla branch {scylla_backend_branch_name} in manager defaults"
+
+    known_distro_dict = {Distro.CENTOS7: "centos7",
```

Comment:
here it appears again... it should be used in an external place, or even the `Distro.version` itself